guides/source/upgrading_ruby_on_rails.md

```diff
@@ -169,81 +169,94 @@ more meaningful preview images. Previously the first frame of the video would be
 and that caused problems if the video faded in from black. This change requires
 FFmpeg v3.4+.
 
-### ActiveStorage variant processor changed to :vips
+### ActiveStorage default variant processor changed to `:vips`
 
-Image transformation will now use libvips instead of ImageMagick. This will reduce
+For new apps, image transformation will use libvips instead of ImageMagick. This will reduce
 the time taken to generate variants as well as CPU and memory usage, improving response
-times in apps that rely on active storage to serve their images. 
+times in apps that rely on active storage to serve their images.
```

Comment:
Not sure if this copy is better than before, we can  retain the one before.

Reply:
ooops, that's what happens when you use the github editor!